derp/derp_server.go

```diff
@@ -120,6 +120,7 @@ type Server struct {
 	multiForwarderCreated    expvar.Int
 	multiForwarderDeleted    expvar.Int
 	removePktForwardOther    expvar.Int
+	averageQueueDuration     expvar.Float
```

Comment:
in what unit? ah, I see it's ms below. put it in the var name `avgQueueDurationMillis` or as a comment `// in milliseconds`